README.md

```diff
@@ -118,236 +120,6 @@ exact matches with those from [LightGBM.](https://lightgbm.readthedocs.io/en/lat
 Not all of these are necessarily supported but see the guide for detailed explanations of what these
```

Comment:
This whole section can be removed and its corresponding part in the table of contents.

Reply:
Remove the "Export" section and its table of contents.